_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,1292 @@
+  to significant profit impacts.
+
+{:#bip3}
+- **BIP3 and the BIP process:** In 2025, work on the Updated BIP Process
+  advanced significantly. BIP3 was [assigned][news341 bip3 assigned] a number in
+  January, published in February, and advanced to Proposed in April. Further
+  review was followed by a few more tweaks, in which SPDX License Expressions
+  were introduced, some Preamble headers were updated, and several clarifications
+  were worked into the proposals. In November, Murch [motioned to
+  activate][news382 motion to activate bip3] the proposal, by requesting that
+  readers review the proposal within another four weeks and comment on whether
+  BIP3 should be activated. A flurry of subsequent review resulted in a few more
+  improvements and the reversion of controversial guidance disallowing the use of
+  LLMs in crafting BIPs. As the year closes out, all review has been addressed,
+  and BIP3 is [seeking rough consensus][bip3 feedback addressed] for activation
+  again.
+
+{:#kernelapi}
+- **Bitcoin Kernel C API introduced:** [Bitcoin Core #30595][news380 kernel] introduced a C
+  header that serves as an API for [`bitcoinkernel`][Bitcoin Core #27587],
+  enabling external projects to interface with Bitcoin Core’s block validation
+  and chainstate logic via a reusable C library. Currently, it is limited to
+  operations on blocks and has feature parity with the now-defunct
+  `libbitcoin-consensus` (see [Newsletter #288][news288 lib]).
+
+  Use cases for `bitcoinkernel` include alternative node implementations, an
+  Electrum server index builder, a [silent payment][topic silent payments]
+  scanner, a block analysis tool, and a script validation accelerator, among
+  others. Several language bindings are in development, including for
+  [Rust][kernel rust], [Go][kernel go], [JDK][kernel jdk], [C#][kernel csharp],
+  and [Python][kernel python].
+
+<div markdown="1" class="callout" id="optech">
+
+## Summary 2025: Bitcoin Optech
+
+In Optech's eighth year, we published 50 weekly [newsletters][] and this
+Year-in-Review special.  Altogether, Optech published over 80,000 English words
+about Bitcoin software research and development this year, the rough equivalent
+of a 225-page book.
+
+Each newsletter and blog post was translated into Chinese, French, and Japanese,
+with other languages also receiving translations, for a total of over 150
+translations in 2025.
+
+In addition, every newsletter this year was accompanied by a [podcast][]
+episode, totaling over 60 hours in audio form and over 500,000 words in
+transcript form.  Many of Bitcoin's top contributors were guests on the show,
+some of them on more than one episode, with a total of 75 different unique
+guests in 2025:
+
+- 0xB10C
+- Abubakar Sadiq Ismail (x3)
+- Alejandro De La Torre
+- Alex Myers
+- Andrew Toth
+- Anthony Towns
+- Antoine Poinsot (x5)
+- Bastien Teinturier (x3)
+- Bob McElrath
+- Bram Cohen
+- Brandon Black
```

Comment:
I couldn’t believe and had to verify that Brandon actually only joined us once this year, but it seems to be accurate.